javascript-data-model/order-history.js

```diff
@@ -0,0 +1,49 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    totalPrice: '$34.00',
+    expectedDelivery: 'August 8, 2020',
+    itemCategory: 'book',
+    returnWindowClosed: 'September 7, 2020',
+    itemName: 'JavaScript for impatient programmers',
+    shipTo: 'JS Masher',
+    orderNumber: '114-3941689-8772232',
+    priceBeforeTax: '$31.55'
+
+  },
```

Comment:
Now all the data for the item in the order is intermixed with the data about the order itself, which is confusing. You should organize the data for each item and store it into a property on the object representing the order.